### PR TITLE
Support for passing of styles in prop

### PR DESCRIPTION
**Pull request by narendrashetty**
This supports passing an optional `styles` prop to `Geosuggest`.
This would support overriding/adding styles to the various components.
Syntax: 
```
<Geosuggest
  style={{
    'input': {
      'borderColor': '#000'
    },
    'suggests': {
      'borderColor': '#000'
     },
    'suggestItem': {
      'borderColor': '#000'
      'borderWidth': 1
    }
  }}
/>
```

This PR is in reference with the issue #109

### CODE REVIEW

File: src/prop-types.js [hunk 1]
```diff
@@ -25,5 +25,6 @@ export default {
   onChange: React.PropTypes.func,
   skipSuggest: React.PropTypes.func,
   getSuggestLabel: React.PropTypes.func,
-  autoActivateFirstSuggest: React.PropTypes.bool
+  autoActivateFirstSuggest: React.PropTypes.bool,
+  style: React.PropTypes.object
```
Comment (ro-ka) [inline src/prop-types.js]: Define what kind of object as mentioned in the [React docs](https://facebook.github.io/react/docs/reusable-components.html):

```js
// An object taking on a particular shape
optionalObjectWithShape: React.PropTypes.shape({
  color: React.PropTypes.string,
  fontSize: React.PropTypes.number
}),
```

**Comment by ro-ka** [comment]
Thanks for the PR! Just a minor thing.

**Comment by narendrashetty** [comment]
@ro-ka made the changes mentioned.

**Comment by ro-ka** [comment]
Perfect, thanks!

**Comment by ro-ka** [comment]
Just released in version `1.24.0`.

**Pull request merged**